apps/google_maps/test/aws_location_test.exs

```diff
@@ -0,0 +1,51 @@
+defmodule AWSLocationTest do
+  use ExUnit.Case, async: true
+
+  # BBox copied from GoogleMaps.Geocode @bounds, should import later
+  @default_options %{
+    FilterBBox: [-71.9380, 41.3193, -69.6189, 42.8266]
+  }
+
+  # TODO: Move index, urls, etc to config
+  @place_index "dotcom-dev-here"
+  @client %AWS.Client{
```

Comment:
Oof, `aws` does not appear to support that. Some poking around its repo's issues suggests that [`aws_credentials`](https://github.com/aws-beam/aws_credentials) might be able to be leveraged for this purpose -- @paulswartz do you have any experience with that library?

Reply:
I do not. I have created new requests for `ExAws` so that's my normal workflow: see https://gitlab.com/paulswartz/kinesis_stage/-/blob/main/lib/kinesis_stage.ex#L32-45 for an example.